packages/components/src/components/Button/tests/Button.ct.tsx

```diff
@@ -0,0 +1,59 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { IconIcon } from '@frontify/fondue-icons';
+import { expect, test } from '@playwright/experimental-ct-react';
+
+import { Button } from '../Button';
+
+const BUTTON_TEXT = 'Frontify';
+
+test.use({ viewport: { width: 500, height: 500 } });
+
+test('should render without error', async ({ mount }) => {
+    const component = await mount(<Button>{BUTTON_TEXT}</Button>);
+    await expect(component).toContainText(BUTTON_TEXT);
+});
+
+test('should render in positive medium and with only text.', async ({ mount }) => {
+    const component = await mount(
+        <Button style="positive" size="medium">
+            {BUTTON_TEXT}
+        </Button>,
+    );
+    await expect(component).toBeVisible();
+    await expect(component).toContainText(BUTTON_TEXT);
+});
+
+test('should react on Click', async ({ mount }) => {
+    const component = await mount(
+        <Button style="positive" size="medium">
+            {BUTTON_TEXT}
+        </Button>,
+    );
+    await expect(component).toBeVisible();
+    await expect(component).toContainText(BUTTON_TEXT);
+});
+
+test('should render in positive medium and with only icon.', async ({ mount }) => {
+    const component = await mount(<Button style="positive" size="medium" icon={<IconIcon />} />);
+    await expect(component).toBeVisible();
+    await expect(component.locator('css=span>svg')).toBeVisible();
+});
+
+test('should render in positive medium and with only only an icon and fully rounded.', async ({ mount }) => {
+    const component = await mount(<Button style="positive" size="medium" icon={<IconIcon />} rounding="full" />);
+    await expect(component).toBeVisible();
+    await expect(component).toHaveClass(/tw-rounded-full/);
+    await expect(component.locator('css=span>svg')).toBeVisible();
+});
+
+test('should render in positive medium with an icon and text.', async ({ mount }) => {
+    const component = await mount(
+        <Button style="positive" size="medium" icon={<IconIcon />}>
+            {BUTTON_TEXT}
+        </Button>,
+    );
+    await expect(component).toBeVisible();
+    await expect(component).toHaveText(BUTTON_TEXT);
+    await expect(component.locator('css=span>svg')).toBeVisible();
```

Comment:
Can you use the icon test id or name instead of `css=span>svg`? (Same for the other tests)

Reply:
changed